### ISSUE TITLE
issues with extg

### ISSUE THREAD

**Issue by bereziat** [heimdali/heimdali#57]
Some problems remains in closed issue #33:
- with incorrect values (out of the domain) for options -x -y -z -v -ix -iy -iz -iv, the command crashed instead of return an error message
- option -xpas without effect (extg -xpas 2 lena.inr | par  -x should return -x 128, but return -x 256
- options -ypas and -zpas crashe, -vpas untested.

**Comment by dfroger** [comment]
Just by the way original `extg` seems bugged.

```
extg -iz 6 -z 3 -zpas 2 -iy 5 -y 2 -ypas 3 -ix 4 -x 3 -xpas 3 -iv 2 -v 2 -vpas 5  $HEIMDALI_DATA_DIR/imtest_z15_y14_x13_c12.inr | tpr -l 6
```

produce:

```
plan 1, ligne 1
5431 5436 5461 5466 5491 5496
plan 1, ligne 2
5406 5407 5408 5409 5410 5411
plan 2, ligne 1
7431 7436 7461 7466 7491 7496
plan 2, ligne 2
7406 7407 7408 7409 7410 7411
plan 3, ligne 1
9431 9436 9461 9466 9491 9496
plan 3, ligne 2
9406 9407 9408 9409 9410 9411
```

instead of:

```
plan 1, ligne 1
5431 5436 5461 5466 5491 5496
plan 1, ligne 2
5731 5736 5761 5766 5791 5796
plan 2, ligne 1
7431 7436 7461 7466 7491 7496
plan 2, ligne 2
7731 7736 7761 7766 7791 7796
plan 3, ligne 1
9431 9436 9461 9466 9491 9496
plan 3, ligne 2
9731 9736 9761 9766 9791 9796
```

or I missunderstand something.

**Comment by dfroger** [comment]
Fixed with 5fa92f8

**Comment by bereziat** [comment]
Thank for this return. I performed some tests and effectively, extg -ypas appears to be bugged.
This is great Heimdali version is not :).
